### DISCUSSION TITLE
4K requests not showing

### DISCUSSION THREAD

**Discussion by v1ct0r93** [sct/overseerr#1590] [Help]
When I request a movie in 4K it does not show a purple clock in the right hand corner of the movie poster. Only when I click on the movie it says that is has been requested. I had multiple people who requested a movie which I already requested in 4k.

Is there a feature that I missed which combines these requests?

**Comment by TheCatLady** [comment]
@v1ct0r93 I recommend joining our Discord server (https://discord.gg/PkCWJSeCk7) and posting in the #support channel for help!
